src/struct_mv/CMakeLists.txt

```diff
@@ -61,9 +61,12 @@ if (HYPRE_USING_GPU)
     struct_data.c
     struct_innerprod.c
     struct_matmult.c
+    struct_matmult_core.c
```

Comment:
This is already correct on recmat.  Do you need to do a merge maybe?

Reply:
Interesting... This shows up as a new change to me. Also, recmat-dev-3 seems up-to-date with recmat

![Screenshot from 2025-05-20 19-26-17](https://github.com/user-attachments/assets/4fe1666b-0def-4ffd-8636-003a5608f7ad)

Reply:
Oh ok! That makes sense, something to keep in mind...